drfutils/serializers.py

```diff
@@ -68,3 +87,13 @@ def __init__(self, *args, **kwargs):
                     self.fields[field_name] = serializer_class(
                         context=self.context
                     )
+
+
+def SerializerBuilder(serializer_class, fields):
```

Comment:
Not sure about this name. Alternative names include `SerializerLimiter`, `SerializerFieldSubset`...